### PR TITLE
Zaylon/working

### PR DESCRIPTION
**Pull request by zaylon10**

### CODE REVIEW

File: extensions.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#get user input
+text = input("File name: ")
+#convert input to lower with .lower string method
+text1 = text.lower()
+#create function that involves branching/ produces desired outcome w/ paramater
+def mediatype(user):
+    if ".jpeg" in user:
```
Comment (stephenlauck) [inline extensions.py]: Could you use a case statement here?

https://www.geeksforgeeks.org/python-match-case-statement/

**Review by stephenlauck — CHANGES_REQUESTED** [review]

**Comment by sosaeio** [comment]
@zaylon10 what if we do something like this

```python
def mediatype(user):
    user = user.lower().strip() # to handle upper case and remove extra spaces, an edge case if file is called " my_file.jpeg   "
    if user.endswith(".gif"): # checkout endswith https://www.programiz.com/python-programming/methods/string/endswith
        return print("image/gif") # return early when you can
    # add similar conditions for other file extension names
```

File: extensions.py (hunk 1 above)
Comment (zaylon10) [reply extensions.py]: @sosaeio @stephenlauck  Thank you! Both suggestions could end in better code optimization.  I will make the necessary revision as soon as possible.

**Comment by zaylon10** [comment]
code has been refactored @sosaeio @stephenlauck

**Review by zaylon10 — COMMENTED** [review]
refactored extensions.py

**Review by stephenlauck — APPROVED** [review]

**Review by stephenlauck — APPROVED** [review]

**Review by stephenlauck — COMMENTED** [review]
Usually it's good to add a new line at the end of the file. Overall looks like you're learning a lot!

**Pull request merged**